packages/redux/src/contents/actions/factories/fetchCommercePagesFactory.ts

```diff
@@ -39,17 +39,19 @@ const fetchCommercePagesFactory =
     let hash: string | undefined;
 
     try {
+      const { slug, ...queryWithoutSlug } = query;
+
       hash = generateContentHash({
         contentTypeCode: ContentTypeCode.CommercePages,
-        ...query,
+        codes: slug,
```

Comment:
This will make requests for commerce pages containing different query parameters to be indexed on the same entry in the redux store. Is this ok?

Reply:
This only generate an hash differently. The hash for commerce pages need to be the url instead of query params